pinot-broker/src/main/java/com/linkedin/pinot/broker/broker/BrokerServerBuilder.java

```diff
@@ -244,8 +244,12 @@ public void start() throws Exception {
 
   public void stop() throws Exception {
     LOGGER.info("Shutting down Network !!");
-
     _state.set(State.SHUTTING_DOWN);
+    try {
+      Thread.sleep(10000);
```

Comment:
What is the need for the sleep if all we do is close connection pool and shutdown?